draft-irtf-cfrg-frost.md

```diff
@@ -568,14 +549,15 @@ each Signer then runs the following procedure to produce its own signature share
   - nonce_i, pair of Scalar values (d, e) generated in round one.
   - comm_i, pair of Element values (D, E) generated in round one.
   - msg, the message to be signed (sent by the Coordinator).
-  - commitment_list = [(j, D_j, E_j), ...], a list of commitments issued by each signer,
+  - commitment_list = [(j, x, y), ...], a list of commitments issued by each signer,
     where each element in the list indicates the signer index and their
-    two commitment Element values. B MUST be sorted in ascending order
+    two commitment Element values (x, y). This list MUST be sorted in ascending order
     by signer index.
   - participant_list, a set containing identifiers for each signer, similarly of length
     NUM_SIGNERS (sent by the Coordinator).
 
-  Outputs: a signature share z_i and commitment share R_i
+  Outputs: a signature share sig_share and commitment share comm_share, which
+           are Element and Scalar values respectively.
```

Comment:
Shouldn't this be "are Scalar and Element values respectively", if I'm reading this correctly?

Reply:
Oh, yes, duh.